deeprank_gnn/NeuralNet.py

```diff
@@ -208,58 +251,64 @@ def put_model_to_device(self, dataset, Net):
-        if self.task == 'reg':
-            self.model = Net(dataset.get(
-                0).num_features, 1, self.num_edge_features).to(self.device)
+        if self.task == "reg":
+            self.model = Net(
+                dataset.get(0).num_features,
+                1,
+                self.num_edge_features).to(
+                self.device)
 
         # classification mode
-        elif self.task == 'class':
-            self.classes_to_idx = {i: idx for idx,
-                                   i in enumerate(self.classes)}
-            self.idx_to_classes = {idx: i for idx,
-                                   i in enumerate(self.classes)}
+        elif self.task == "class":
+            self.classes_to_idx = {
+                i: idx for idx, i in enumerate(
+                    self.classes)}
+            self.idx_to_classes = dict(enumerate(self.classes))
             self.output_shape = len(self.classes)
 
-            self.model = Net(dataset.get(
-                0).num_features, self.output_shape, self.num_edge_features).to(self.device)
+            self.model = Net(
+                dataset.get(0).num_features,
+                self.output_shape,
+                self.num_edge_features).to(
+                self.device)
 
     def set_loss(self):
         """Sets the loss function (MSE loss for regression/ CrossEntropy loss for classification)."""
-        if self.task == 'reg':
+        if self.task == "reg":
             self.loss = MSELoss()
 
-        elif self.task == 'class':
+        elif self.task == "class":
 
             # assign weights to each class in case of unbalanced dataset
             self.weights = None
-            if self.class_weights == True:
+            if self.class_weights:
                 targets_all = []
                 for batch in self.train_loader:
                     targets_all.append(batch.y)
 
-                targets_all = torch.cat(
-                    targets_all).squeeze().tolist()
+                targets_all = torch.cat(targets_all).squeeze().tolist()
                 self.weights = torch.tensor(
-                    [targets_all.count(i) for i in self.classes], dtype=torch.float32)
-                print('class occurences: {}'.format(self.weights))
+                    [targets_all.count(i) for i in self.classes], dtype=torch.float32
+                )
+                print(f"class occurences: {self.weights}")
                 self.weights = 1.0 / self.weights
                 self.weights = self.weights / self.weights.sum()
-                print('class weights: {}'.format(self.weights))
+                print(f"class weights: {self.weights}")
 
             self.loss = nn.CrossEntropyLoss(
-                weight=self.weights, reduction='mean')
+                weight=self.weights, reduction="mean")
 
     def train(self, nepoch=1, validate=False, save_model='last',
-              hdf5='train_data.hdf5'):
+              hdf5='train_data.hdf5'): # noqa
```

Comment:
`hdf5` argument is not used. Should I remove it?

Reply:
Yes, remove it! I must have forgotten to remove that in my previous pull request.